### ISSUE TITLE
Make hover descriptions (the text that appears at the bottom of the page when a chart is hovered) instead appear above each chart and persist after mouseout

### ISSUE THREAD

**Issue by spasovski** [mozilla/distribution-viewer#146]
This is to make it closer to FHR, probably lower priority.

**Comment by openjck** [comment]
Is this the description text (#93) or the summary text (#94) or both?

**Comment by spasovski** [comment]
Summary - though summary is a weird name for it.

**Comment by openjck** [comment]
Agreed. I'm starting to see that my chosen terminology is rather confusing. :smile:

**Comment by openjck** [comment]
In other words, our hover descriptions should look more like they do in Brendan's charts. They should appear below the chart title once the mouse is moved over the chart and stay there after the mouse moves out.

Here's what it looks like on Brendan's dashboard (I removed the line from the chart to keep the data private):

![](https://cloud.githubusercontent.com/assets/933396/23465350/ff2bbef6-fe65-11e6-8624-3b40ef9edae4.png)
